core/templates/pages/exploration-editor-page/services/version-history.service.ts

```diff
@@ -59,6 +59,13 @@ export class VersionHistoryService {
   fetchedMetadata: (ExplorationMetadata | null) [] = [];
 
   fetchedCommitterUsernames: string[] = [];
+
+  // The two below variables represent the current index of the version history
+  // list which the user is navigating i.e. index value for the lists
+  // fetchedStateVersionNumbers, fetchedMetadataVersionNumbers etc. These
+  // variables are initialized as zero inside the resetStateVersionHistory and
+  // resetMetadataVersionHistory functions when a state editor or the
+  // exploration settings tab is opened by a user.
```

Comment:
Add (if true): "This is because the item that is shown to a user by default is the first item in each of these lists."

Reply:
Done.